src/sentry/static/sentry/app/actionCreators/projects.jsx

```diff
@@ -34,3 +36,53 @@ export function loadStats(api, params) {
 export function setActiveProject(project) {
   ProjectActions.setActive(project);
 }
+
+export function removeProject(api, orgId, project) {
+  let endpoint = `/projects/${orgId}/${project.slug}/`;
+
+  ProjectActions.removeProject(project);
+  let req = api
```

Comment:
nit: just return here, no need for a variable

Reply:
Ah, this is related to your comments about rethrowing - I generally return the request promise, but here it's returning the promise of the handler chains. I suppose it's better to just explicitly re-throw in rejection handler because this is such a subtle difference.